Testreglar/2.2.1/Nett/nett-2.2.1a.json

```diff
@@ -0,0 +1,335 @@
+{
+	"namn": "Nett-2.2.1a Det er mogleg å slå av, justere eller forlenge tidsavgrensingar 2023",
+	"id": "nett-2.2.1a",
+	"testlabId": 506,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<p>Det er mulig for brukeren å</p><ul><li>Slå av tidsavgrensningen, senest 20 sekunder før den skjer, ved å gjøre en enkel handling<ul><li>Mekanismen for å slå av skal ligge før eller rett ved siden av innholdet som inneholder tidsavgrensninger.</li><li>Slå av innebærer å slå av tidsavgreningen permanent for det aktuelle besøket eller</li></ul></li><li>Justere tidsavgrensningen med minst 10 ganger standard varighet.<ul><li>Mekanismen for å justere skal ligge før eller rett ved siden av innholdet som inneholder tidsavgrensninger eller</li></ul></li><li>Forlenge tidsavgrensningen senest 20 sekunder før den inntreffer, ved å gjøre en enkel handling. Det skal være mulig å forlenge med minst 10 ganger standard varighet.<ul><li>Mekanismen for å forlenge skal ligge før eller rett ved siden av innholdet som inneholder tidsavgrensninger.</li></ul></li></ul>",
```

Comment:
Det er bare en av disse som må være oppfylt,

For each time limit that is set by the content, at least one of the following is true:

Det må gå tydeligere frem i kravet til samsvar.

Reply:
@Sivzilla82 kan du fikset krav til samsvar?

Reply:
I og med at det står eller mellom kvar alternativ, så brurde det vera greitt. 

Når det er sagt har eg no oppdatert innledninga for krav til samsvar slik at det no står: **For hver tidsbegrensning som er angitt av innholdet, er det mulig for brukeren å gjøre minst ett av følgende**. I tillegg har eg fjerna eller mellom alternativa.